modules/transdecoder/longorf/main.nf

```diff
@@ -0,0 +1,39 @@
+process TRANSDECODER_LONGORF {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::transdecoder=5.5.0" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+    'https://depot.galaxyproject.org/singularity/transdecoder:5.5.0--pl5262hdfd78af_4' :
+    'quay.io/comp-bio-aging/transdecoder' }"
+
+    input:
+    tuple val(meta), path(fasta)
+
+    output:
+    tuple val(meta), path("${meta.id}/*.pep")                          , emit: pep
+    tuple val(meta), path("${meta.id}/*.gff3")                         , emit: gff
+    tuple val(meta), path("${meta.id}/*.cds")                          , emit: fna
```

Comment:
You're renaming some, but not all, channels from file suffixes. Personally, I like `fna` for fasta nucleotides, but also `faa` for amino acids, and `gff` makes sense, but I'm not sure what is the best practice.